### PR TITLE
Fix PiP aspect ratio issue

### PR DESCRIPTION
**Pull request by MagicalMeghan**
Fixing the PiP issue where the app crashes when the app starts playing a video from landscape mode.

### CODE REVIEW

**Comment by SigmanZero** [comment]
Tested with a Pixel Fold - not seeing a crash when trying landscape playback now, thank you!

File: app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt [hunk 1]
```diff
@@ -179,7 +180,17 @@ private fun VideoPlayer(
                 // set source rect hint, aspect ratio and remote actions
                 val sourceRect = layoutCoordinates.boundsInWindow().toAndroidRectF().toRect()
                 builder.setSourceRectHint(sourceRect)
-                builder.setAspectRatio(Rational(sourceRect.width(), sourceRect.height()))
+                var aspectRatio = Rational(sourceRect.width(), sourceRect.height())
+                val upperAspectRatioBound = Rational(2.39.toInt(), 1)
+                val lowerAspectRatioBound = Rational(1, 2.39.toInt())
```
Comment (neelanshsahai) [inline app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt:185]: 2.39.toInt() will always be 2. We can say Rational(239, 100) and Rational(100, 239) for exact values.

**Review by neelanshsahai — CHANGES_REQUESTED** [review]
Minor changes requested

File: app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt (hunk 1 above)
Comment (MagicalMeghan) [reply app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt:185]: makes sense!!

**Review by wu-perry — APPROVED** [review]
LGTM

**Comment by hwwang-google** [comment]
LGTM

**Comment by SigmanZero** [comment]
Tested with both a landscape and a portrait video, in both landscape and portrait device orientations - fix LGTM, thank you!

File: app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt [hunk 2]
```diff
@@ -174,12 +174,14 @@ private fun VideoPlayer(
...
-
             if (shouldEnterPipMode) {
-                // set source rect hint, aspect ratio and remote actions
-                val sourceRect = layoutCoordinates.boundsInWindow().toAndroidRectF().toRect()
-                builder.setSourceRectHint(sourceRect)
-                builder.setAspectRatio(Rational(sourceRect.width(), sourceRect.height()))
+                // set source rect hint, aspect ratio
+                if (player != null && player.videoSize.width > 0 && player.videoSize.height > 0) {
+                    val sourceRect = layoutCoordinates.boundsInWindow().toAndroidRectF().toRect()
+                    builder.setSourceRectHint(sourceRect)
```
[the remark this answers is not in the document]
Comment (MagicalMeghan) [reply app/src/main/java/com/google/android/samples/socialite/ui/player/VideoPlayerScreen.kt:181]: I had a similar thought - it is definitely worth looking into!

**Pull request merged**